src/operator_repo/utils.py

```diff
@@ -110,6 +110,58 @@ def load_yaml(path: Path) -> Any:
     return _load_yaml_strict(_find_yaml(path))
 
 
+def _load_multidoc_yaml_strict(path: Path) -> List[Dict[str, Any]]:
```

Comment:
Thanks, I didn't realize the specifics of the yaml format. I changed the return value to `Any` with comment for users to be aware of this and handle the result accordingly.
For the typing, it was originally meant to support older versions of Python, but thanks to you I realized the default `list, dict` are usable since Py3.9.